hack/components/bump-linux-bridge.sh

```diff
@@ -49,13 +56,65 @@ COPY --from=builder ${LINUX_BRIDGE_PATH}/bin/tuning ${LINUX_BRIDGE_TAR_CONTAINER
 RUN sha256sum ${LINUX_BRIDGE_TAR_CONTAINER_DIR}/bridge >${LINUX_BRIDGE_TAR_CONTAINER_DIR}/bridge.checksum
 RUN sha256sum ${LINUX_BRIDGE_TAR_CONTAINER_DIR}/tuning >${LINUX_BRIDGE_TAR_CONTAINER_DIR}/tuning.checksum
 EOF
-    ${OCI_BIN} build -t ${LINUX_BRIDGE_IMAGE_TAGGED} .
+}
+
+check_and_create_docker_builder() {
+    existing_builder=$(docker buildx ls | grep -w "$DOCKER_BUILDER" | awk '{print $1}' || true)
+    if [ -n "$existing_builder" ]; then
+        echo "Builder '$DOCKER_BUILDER' already exists. Using existing builder."
+        docker buildx use "$DOCKER_BUILDER"
+    else
+        echo "Creating a new Docker Buildx builder: $DOCKER_BUILDER"
+        docker buildx create --driver-opt network=host --use --name "$DOCKER_BUILDER"
+    fi
+}
+
+build_docker_image() {
+    docker buildx build --platform "${PLATFORMS}" --build-arg BUILD_ARCH="$ARCH" -t "${LINUX_BRIDGE_IMAGE_TAGGED}" . --push
+    docker buildx rm "$DOCKER_BUILDER"
+}
+
+build_podman_image() {
+    podman manifest rm "${LINUX_BRIDGE_IMAGE_TAGGED}" || true
+    podman rmi "${LINUX_BRIDGE_IMAGE_TAGGED}" || true
+    podman manifest create "${LINUX_BRIDGE_IMAGE_TAGGED}"
+
+    for platform in "${PLATFORM_LIST[@]}"; do
+        podman build --no-cache --build-arg BUILD_ARCH="$ARCH" --platform "$platform" --manifest "${LINUX_BRIDGE_IMAGE_TAGGED}" .
+    done
+}
+
+modify_dockerfile_for_platform_and_architecture() {
+    local dockerfile="$1"
+    # Modify Dockerfile to set platform and architecture
+    sed -i 's|^FROM registry.access.redhat.com/ubi8/ubi-minimal AS builder$|FROM --platform=linux/${BUILD_ARCH} registry.access.redhat.com/ubi8/ubi-minimal AS builder|' "$dockerfile"
+    sed -i 's|RUN GOFLAGS=-mod=vendor ./build_linux.sh|RUN GOFLAGS=-mod=vendor GOARCH=${TARGETARCH} GOOS=${TARGETOS} ./build_linux.sh|' "$dockerfile"
+    sed -i 's/^FROM registry.access.redhat.com\/ubi8\/ubi-minimal AS final$/FROM --platform=linux\/${TARGETARCH} registry.access.redhat.com\/ubi8\/ubi-minimal AS final/' "$dockerfile"
+}
+
+(
+    cd ${LINUX_BRIDGE_PATH}
+    create_dockerfile
+    modify_dockerfile_for_platform_and_architecture "Dockerfile"
+    (
+        if [[ "${OCI_BIN}" == "docker" ]]; then
+            check_and_create_docker_builder
+            build_docker_image
+        elif [[ "${OCI_BIN}" == "podman" ]]; then
+            build_podman_image
+        else
+            echo "Invalid OCI_BIN value. It must be either 'docker' or 'podman'."
+            exit 1
+        fi
+    )
 )
 
 echo 'Push the image to KubeVirt repo'
 (
-    if [ ! -z ${PUSH_IMAGES} ]; then
-        ${OCI_BIN} push "${LINUX_BRIDGE_IMAGE_TAGGED}"
+    if [ "${OCI_BIN}" == "podman" ]; then
+        if [ ! -z ${PUSH_IMAGES} ]; then
+            podman manifest push "${LINUX_BRIDGE_IMAGE_TAGGED}"
+        fi
```

Comment:
please move it after `build_podman_image`

Reply:
Done